app/src/app/page.tsx

```diff
@@ -89,51 +113,81 @@ export default function Home() {
+					>
+						<div className="text-center mb-4">
+							<h2 className="text-lg font-semibold mb-2">今日のお題</h2>
+							<p className="text-sm text-gray-600">撮影してスコアを競おう！</p>
+						</div>
+						<h1 className="text-3xl font-bold text-center mb-4">
+							{todayAssignment?.english}
+						</h1>
+						<div className="flex justify-center w-full">
+							<Button
+								variant="default"
+								className="flex items-center justify-center w-3/4 bg-gray-800 hover:bg-gray-700 text-white py-6 space-x-2"
+								onClick={() => router.push("/camera")}
+							>
+								<div className="w-6 h-auto">
+									<PhotoCameraIcon />
+								</div>
+								<span className="text-lg font-semibold">写真を撮る</span>
+							</Button>
 						</div>
 					</Card>
-				) : (
-					<div className="text-lg w-full">
-						{latestAssignment?.assignTime && (
-							// fixme [0]番目を参照しているがお題ごとに可変的にする必要あり。
-							<Timer assignTime={latestAssignment?.assignTime} />
-						)}
-					</div>
 				)}
-				<Card
-					className="flex flex-col items-center justify-around aspect-square w-full p-6 backdrop-blur-sm"
-					style={{
-						background:
-							"linear-gradient(90deg, rgba(255, 145, 109, 0.56) 0%, rgba(255, 90, 170, 0.44) 51%, rgba(139, 166, 255, 0.61) 100%)",
-					}}
-				>
-					<div className="text-center mb-4">
-						<h2 className="text-lg font-semibold mb-2">今日のお題</h2>
-						<p className="text-sm text-gray-600">撮影してスコアを競おう！</p>
-					</div>
-					<h1 className="text-3xl font-bold text-center mb-4">
-						{latestAssignment?.english}
-					</h1>
-					<div className="flex justify-center w-full">
-						<Button
-							variant="default"
-							className="flex items-center justify-center w-3/4 bg-gray-800 hover:bg-gray-700 text-white py-6 space-x-2"
-							onClick={() => router.push("/camera")}
-						>
-							<div className="w-6 h-auto">
-								<PhotoCameraIcon />
+				{isAnsweredAll && (
+					<Card className="w-full max-w-md bg-white overflow-hidden">
+						<div className="p-8 text-center">
+							<LuCheckCircle className="w-20 h-20 text-green-500 mx-auto mb-6" />
+							<h2 className="text-3xl font-bold mb-4 text-gray-800">
+								全課題完了
+							</h2>
+							<p className="text-gray-600 mb-8">
+								おめでとうございます。全ての課題をクリアしました！
+							</p>
+							<div className="bg-gray-100 rounded-2xl p-6 mb-8">
+								<h3 className="text-xl font-semibold text-gray-700 mb-2">
+									最終スコア
+								</h3>
+								<p className="text-4xl font-bold text-purple-600">
+									{highestScore}点
+								</p>
 							</div>
-							<span className="text-lg font-semibold">写真を撮る</span>
-						</Button>
-					</div>
-				</Card>
+							<Button
+								onClick={() => router.push("/ranking")}
+								className="w-full bg-gradient-to-r from-pink-500 to-purple-500 text-white font-semibold py-3 px-6 rounded-full hover:from-pink-600 hover:to-purple-600 transition duration-300"
+							>
+								結果を確認する
+								<LuArrowRight className="ml-2 w-5 h-5" />
+							</Button>
```

Comment:
```suggestion

							<Button className="w-full bg-gradient-to-r from-pink-500 to-purple-500 text-white font-semibold py-3 px-6 rounded-full hover:from-pink-600 hover:to-purple-600 transition duration-300">
								結果を確認する
								<Link href="/ranking">
									<LuArrowRight className="ml-2 w-5 h-5" />
								</Link>
```

next/linkの形式にした方がいいかも